languages/kurdish/pack/dictionary/prebuilt/websites.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think you can remove this file too, since Kurdish does not have latin characters the words on this file will never be used.